lib/web/files.go

```diff
@@ -130,6 +161,13 @@ func (f *fileTransfer) upload(req fileTransferRequest, httpReq *http.Request) er
 		return trace.Wrap(err)
 	}
 
+	if req.webauthn != "" {
+		err = f.issueSingleUseCert(req.webauthn, httpReq, tc)
```

Comment:
At first it wasn't clear to me what happens to the single use cert that gets issued. Maybe a godoc on `issueSingleUseCert` that explains that it configures `tc` to use the new cert would help.